bin/cml-tensorboard-dev.js

```diff
@@ -32,51 +33,52 @@ const run = async (opts) => {
   await fs.writeFile(`${path}/uploader-creds.json`, credentials);
 
   // launch  tensorboard on background
-  const tb_path = await exec('which tensorboard');
   const help = await exec('tensorboard dev upload -h');
   const extra_params_found =
     (name || description) && help.indexOf('--description') >= 0;
   const extra_params = extra_params_found
     ? `--name "${name}" --description "${description}"`
     : '';
-  const command = `python -u ${tb_path} dev upload --logdir ${logdir} ${extra_params}`;
 
-  const proc = spawn(command, { detached: true, shell: true });
-
-  proc.stderr.on('data', (data) => {
-    data && console.error(data.toString('utf8'));
+  const command = `tensorboard dev upload --logdir ${logdir} ${extra_params}`;
+  const stdout_path = tempy.file({ extension: 'log' });
+  const stdout_fh = await fs.open(stdout_path, 'a');
```

Comment:
what `_fh` stands for?

Reply:
file_handler

Reply:
please keep it if it is a standard term in JS? I get used to file descriptor (fd).

Reply:
changed to _fd